cms/src/main/resources/api-specification/codegen-templates/params_table.mustache

```diff
@@ -6,6 +7,84 @@
                 <th>Description</th>
             </tr>
         </thead>
-        {{#block "request-param-table"}}{{/block}}
+        {{#this}}
+        <tr>
+            <td><code class="nhsd-!t-font-family-mono">{{baseName}}</code></td>
+            <td>
+                {{#or dataType datatype dataFormat}}
+                    <p class="nhsd-t-body">
+                        {{#response}}
+                            {{#datatype}}{{! Lowercase on io.swagger.codegen.v3.CodegenProperty}}
```

Comment:
whoa what is this syntax hahaha - is this like a static method reference?

Reply:
Do you mean this bit?:
```
{{! Lowercase on io.swagger.codegen.v3.CodegenProperty}}
```
That is simply a Handlebars/Mustache comment. `{{! ... }}` ensures that the content between the curly braces will not be rendered in any way.

I've now rephrased the two comments to be less ambiguous:
```
{{! This property is lowercased on io.swagger.codegen.v3.CodegenProperty}}
{{! This property is camelCased on io.swagger.codegen.v3.CodegenParameter}}
```

Reply:
ahh got it :+1: